src/stylesheets/abstracts/variables.scss

```diff
@@ -0,0 +1 @@
+// Color Variables
```

Comment:
This file needs to be named `_variables`

Without the `_`, it's being compiled into its own .css file in `/public/assets`